### PR TITLE
Update json for nf-nomad 0.2.0 release

### PR DESCRIPTION
**Pull request by abhi18av**
Hi team 👋 

We're all set for another release for `nf-nomad` plugin, could someone please take this forward 🚀

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -3082,7 +3082,14 @@
...
+        "version": "0.2.0",
+        "date": "2024-08-28T16:24:25.597979509Z",
+        "url": "https://github.com/nextflow-io/nf-nomad/releases/download/0.2.0/nf-nomad-0.2.0.zip",
+        "requires": ">=24.04.0",
+        "sha512sum": "fa67b5fc0a86b1562dbe3c91833077f3a2551fa5ebe6f4e22fefe4179fe0535b14019ae8b7283100415e0eab453cb35eac08f7671e85771b29c9349d8aeba24d"
       }
     ]
   }
-]
+]
```
Comment (bentsherman) [inline plugins.json:3095]: See if you can revert this change

Comment (abhi18av) [reply plugins.json:3095]: Hey Ben, I check it again and I think that `]` is needed - it's just pushed down because of the new JSON object insertion for the release no?

Comment (bentsherman) [reply plugins.json:3095]: I think it has to do with how you saved the file locally, it removed the EOF. I couldn't find a way to revert it on github without adding an extra empty line

Comment (abhi18av) [reply plugins.json]: Gotcha, should be fixed now 👍
